frontend/src/routes/ClusterManagement/Clusters/components/ProgressStepBar.tsx

```diff
@@ -95,20 +100,50 @@ export function ProgressStepBar() {
                 if (prehooks) {
                     prehookStatus = StatusType.healthy
                 } else {
-                    prehookStatus = StatusType.pending
+                    prehookStatus = StatusType.healthy
                 }
         }
 
+        const provisionStatus: string[] = [
+            ClusterStatus.creating,
+            ClusterStatus.provisionfailed,
+            ClusterStatus.importing,
+            ClusterStatus.importfailed,
+            ClusterStatus.posthookjob,
+            ClusterStatus.posthookfailed,
+        ]
+        const posthookJobStatus: string[] = [ClusterStatus.posthookjob, ClusterStatus.posthookfailed]
+
         const steps: ProgressTrackerStep[] = [
             {
                 statusType: prehookStatus,
                 statusText: t('status.prehook.text'),
                 statusSubtitle: prehooks ? t(`status.subtitle.${prehookStatus}`) : t('status.subtitle.nojobs'),
+                // will render link when prehook job url is defined or when there are no job hooks setup
+                ...((prehooks || (!prehooks && !posthooks)) && {
+                    link: {
+                        linkName: !prehooks && !posthooks ? t('status.link.info') : t('status.link.logs'),
+                        // TODO: add ansible documentation url
+                        linkUrl:
+                            !prehooks && !posthooks
+                                ? 'https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.2/'
+                                : latestJobs.prehook?.status?.ansibleJobResult?.url
+                                ? latestJobs.prehook?.status?.ansibleJobResult?.url
+                                : '',
+                        isDisabled: !latestJobs.prehook?.status?.ansibleJobResult?.url,
+                    },
+                }),
             },
             {
                 statusType: creatingStatus,
                 statusText: t('status.install.text'),
                 statusSubtitle: t(`status.subtitle.${creatingStatus}`),
+                ...(provisionStatus.includes(cluster?.status ?? '') && {
```

Comment:
I think the `??` is not needed because the cluster is always defined with a status